wranglerjs/index.js

```diff
@@ -65,31 +65,41 @@ fetchCompileWasmTemplatePlugin.fn = function(compilation) {
   plugin.apply(mainTemplate);
 };
 
-compiler.run((err, stats) => {
+let lastHash = "";
+const compilerCallback = (err, stats) => {
   if (err) {
     throw err;
   }
 
-  const assets = stats.compilation.assets;
-  const jsonStats = stats.toJson();
-  const bundle = {
-    wasm: null,
-    script: "",
-    errors: jsonStats.errors
-  };
-
-  const wasmModuleAsset = Object.keys(assets).find(filterByExtension("wasm"));
-  const jsAssets = Object.keys(assets).filter(filterByExtension("js"));
-  const hasWasmModule = wasmModuleAsset !== undefined;
-
-  bundle.script = jsAssets.reduce((acc, k) => {
-    const asset = assets[k];
-    return acc + asset.source();
-  }, "");
-
-  if (hasWasmModule === true) {
-    bundle.wasm = Buffer.from(assets[wasmModuleAsset].source()).toString("base64");
+  if (stats.hash != lastHash) {
```

Comment:
```suggestion
  if (stats.hash !== lastHash) {
```

If both are strings.